hosts

```diff
@@ -4,6 +4,10 @@ ny1.hashbang.sh
 sf1.hashbang.sh
 to1.hashbang.sh
 
+[shell_servers]
+ansible_user=root
```

Comment:
In general I like to disable root login via sshd. I'd like to do that for hashbang too but... this. :)